study.md

```diff
@@ -134,15 +134,23 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP is a proctocol used to take with servers on the network. It is a simple
+protocol that issues a request to the server and the server docodes the request
+and sends back the response to the said request. The response will have all
+the necesary resources needed to display you page. HTML, imagines, etc.
```

Comment:
Good it is one protocol we can use to have our client's and server's 'talk' to each other.